src/Symfony/Component/HttpKernel/EventListener/RobotsTagListener.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\EventListener;
+
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+
+/**
+ * Ensure that Dev website never get indexed by search engines.
+ *
+ * @author Gary PEGEOT <garypegeot@gmail.com>
+ */
+class RobotsTagListener
```

Comment:
Could be named `DevRobotsTagListener`? Trying to find a way to make it clear that this is only needed/useful for the dev env.

Reply:
Moreover ... don't you think that `RobotsTagListener` is too specific about _how_ it does its work ... instead of describing _what_ it does? Proposal: `NoIndexListener`, `DevNoIndexListener`, `NoIndexInDevListener`, etc.

Reply:
Maybe **DisallowRobotsIndexListener** or **DisallowIndexListener**?

Reply:
`DebugIndexPreventionListener` sounds good to you?

Reply:
`DefaultRobotsListener` ?

for being component code, IMHO there's no real need to hint about dev-env etc.

Reply:
Renamed `DisallowRobotsIndexingListener`